VisualizationBase/src/Scene.cpp

```diff
@@ -532,6 +532,7 @@ void Scene::setMainViewScalingFactor(qreal factor)
 // Reimplemented in order to detect mouse clicks
 void Scene::mouseMoveEvent(QGraphicsSceneMouseEvent* mouseEvent)
 {
+	lastMousePosition_ = mouseEvent->scenePos();
```

Comment:
Hmm, and just as I wrote that, I see that you actually introduced this field :)

Anyway, I would call it lastMouseHoverPosition_, to make it a bit clearer what it means.